polaris-icons/icons/GaugeMajor.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You should be able to remove the width and height attrs and the viewbox should be `00 20 20`. Probably needs to be re-exported with its full canvas that's 20x20. 

Other things that can be changed:
- clip-rule attrs can be removed
- Only use fills of `#5C5F62`. Replace `fill="black"` with `fill="#5C5F62"`

The errors are a bit tough to read but they can be seen in the [CI check](https://github.com/Shopify/polaris/actions/runs/5133758336/jobs/9236800126?pr=9328)